src/features/content-answers/ContentAnswersOpenButton.scss

```diff
@@ -17,4 +31,28 @@
     &:not(.bdl-is-disabled):hover {
         background-color: $bdl-gray-10;
     }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions {
+        @include BackgroundGradient;
+
+        .bdl-BoxAIIcon {
+            path {
+                fill: $white;
+            }
+        }
+    }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions:hover {
+        @include BackgroundGradientHover;
+    }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions:not(.bdl-is-disabled):focus,
+    &.bdl-ContentAnswersOpenButton--hasQuestions:not(.is-disabled):focus {
+        @include BackgroundGradientHover;
+        @include ButtonOutline;
+    }
+
+    &-tooltip {
+        text-align: center;
+    }
```

Comment:
this wouldn't work because `bdl-ContentAnswersOpenButton-tooltip` is currently a wrapper around the button class `bdl-ContentAnswersOpenButton` so it's not a descendent element. technically `bdl-ContentAnswersOpenButton` should be moved to the root element in SUIT but that probably introduces more headache then it resolves

but two thoughts:
1. do we actually need `text-align: center`? I know it's used in the EUA version but it doesn't follow the pattern we use for the rest of BUIE / Preview to center text in a tooltip
2. `&-tooltip` hurts searchability when searching for `bdl-ContentAnswersOpenButton-tooltip` in GitHub or IDE